### PR TITLE
Implemented api data to movies

### PR DESCRIPTION
**Pull request by AdrianJar**
Implemented most of api data to movies. Still need to add styled, RWD and make solution to genres

### CODE REVIEW

File: src/feature/MovieBrowser/Movies/List/PopularMovies/Loading/index.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react'
```
Comment (KatarzynaZaloba) [inline src/feature/MovieBrowser/Movies/List/PopularMovies/Loading/index.js:1]: image
Great job! The css will be defined later :)

File: src/feature/MovieBrowser/Movies/List/PopularMovies/ErrorBox/index.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React from 'react'
```
Comment (KatarzynaZaloba) [inline src/feature/MovieBrowser/Movies/List/PopularMovies/ErrorBox/index.js:1]: I was unable to see it in action, but well done!

File: src/feature/MovieBrowser/Movies/List/PopularMovies/Data/Rating/index.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import React from 'react'
```
Comment (KatarzynaZaloba) [inline src/feature/MovieBrowser/Movies/List/PopularMovies/Data/Rating/index.js]: image
Now the votes & rating are working!

**Review by KatarzynaZaloba — APPROVED** [review]
I don't see commited MovieDescription, so I had to delete it in my review, because there were alerts in App.js, so I am not sure if this can be the reson why pagination doesn't work :( 
But anyway, GREAT WORK! I hope we can merge it asap then I can connect API for popularpeople page ;)

**Review by Marianna-Weychan — APPROVED** [review]
Really great job Adrian!
I hope that after confirming the pull, the changes will be visible to me so that I can build the logic for downloading a single video

**Pull request merged**